indexing-solr/src/test/java/edu/unc/lib/boxc/indexing/solr/filter/SetContentStatusFilterTest.java

```diff
@@ -218,4 +233,46 @@ public void testIsAssignedThumbnail() throws Exception {
         verify(idb).setContentStatus(listCaptor.capture());
         assertTrue(listCaptor.getValue().contains(FacetConstants.ASSIGNED_AS_THUMBNAIL));
     }
+
+    @Test
+    public void testFileObjectHasAccessSurrogate() throws IOException {
+        Files.write(accessSurrogatePath, List.of("fake image"));
+        when(workObj.getResource()).thenReturn(resc);
+        when(dip.getContentObject()).thenReturn(fileObj);
+        when(fileObj.getResource()).thenReturn(fileResc);
+
+        filter.filter(dip);
+
+        verify(idb).setContentStatus(listCaptor.capture());
+        assertTrue(listCaptor.getValue().contains(FacetConstants.HAS_ACCESS_SURROGATE));
+    }
+
+    @Test
+    public void testFileObjectNoAccessSurrogate() {
+        when(workObj.getResource()).thenReturn(resc);
+        when(resc.hasProperty(Cdr.primaryObject, fileResc)).thenReturn(true);
+        when(dip.getContentObject()).thenReturn(fileObj);
+        when(fileObj.getResource()).thenReturn(fileResc);
+
+        filter.filter(dip);
+
+        verify(idb).setContentStatus(listCaptor.capture());
+        assertTrue(listCaptor.getValue().contains(FacetConstants.NO_ACCESS_SURROGATE));
+    }
+
+    @Test
+    public void testWorkWithFileObjectThatHasAccessSurrogate() throws IOException {
+        Files.write(accessSurrogatePath, List.of("fake image"));
+        when(workObj.getResource()).thenReturn(resc);
+        when(dip.getContentObject()).thenReturn(workObj);
+        when(workObj.getResource()).thenReturn(resc);
+        when(fileObj.getResource()).thenReturn(fileResc);
+        workObj.addMember(fileObj);
+        when(fileObj.getParent()).thenReturn(workObj);
```

Comment:
workObj is a mock, so calling addMember probably isn't doing anything. SetContentStatusFilter doesn't check anything about the files belonging to a work, so there doesn't seem to be anything productive we could do to set this up in a unit test. Maybe we should just remove all the fileObj stuff here and have the test just verifying that works don't get HAS_ACCESS_SURROGATE or NO_ACCESS_SURROGATE

Reply:
done!